skcosmo/feature_selection/_greedy.py

```diff
@@ -0,0 +1,154 @@
+"""
+Sequential feature selection
+"""
+import numbers
+
+import numpy as np
+
+from sklearn.feature_selection._base import SelectorMixin
```

Comment:
We checked it with @ceriottm , in new version here should be `from sklearn.feature_selection import SelectorMixin`

Reply:
Do you not get a warning when you do this import? I was just getting it, so I replaced the import with `from sklearn.feature_selection import SelectorMixin`